.github/workflows/update_genuary.yaml

```
@@ -0,0 +1,50 @@
+name: Update Genuary
+
+on:
+  workflow_dispatch:
+  schedule:
+    # We want '0 9 * 1 *' in Pacific Time, but GitHub Actions doesn't support timezones.
+    # So we use '0 17 * 1 *' in UTC, which is 9 AM in Pacific Time.
+    - cron: '0 17 * 1 *'
```

Allow year-round Genuary submissions by checking every day of the year instead of only the days in the month of January. Reference: <https://crontab.guru/#0_17_*_*_*>.

```suggestion
    - cron: '0 17 * * *'
```